src/main/java/menion/android/whereyougo/network/activity/DownloadCartridgeActivity.java

```diff
@@ -50,9 +50,11 @@
 
 public class DownloadCartridgeActivity extends CustomActivity
         implements NoPasswordDialogFragment.NoPasswordDialogListener {
-    private static final String TAG = "DownloadCartridgeActivity";
+    private static final String TAG = "DownloadCartridgeAct";
```

Comment:
The TAG is mostly used for DEBUG messages. The text is the class name.

Reply:
@bekuno Yes but it can't be longer than 25 characters. If you use it in a `Log.*` statement this will be a critical error.

Reply:
Okay the `TAG` is not used anymore. I just removed it.
@bekuno As a reference: https://developer.android.com/reference/android/util/Log#isLoggable(java.lang.String,%20int)

Reply:
Thanks.